hc/api/tests/test_healthwealth.py

```diff
@@ -0,0 +1,48 @@
+from hc.api.models import ExternalChecks, Check
+from hc.front.forms import ExternalChecksForm
+from hc.test import BaseTestCase
+from time import sleep
+
+
+class HealthWealthTestCase(BaseTestCase):
+    def setUp(self):
+        super(HealthWealthTestCase, self).setUp()
+        self.check = Check()
+        self.external_check = ExternalChecks()
+        self.external_check.save()
+
+    def test_user_can_create_integration(self):
```

Comment:
it's good to implement doc string to state what it's supposed to do.

Reply:
Yeah...lemme fix that.